File: src/main/java/com/tencent/bk/devops/atom/pojo/plugin/PipelineBuildMaterial.java
package com.tencent.bk.devops.atom.pojo.plugin;


import lombok.Data;

@Data
public class PipelineBuildMaterial {
    private String aliasName;
    private String branchName;
    private Integer commitTimes;
    private String newCommitComment;
    private String newCommitId;
    private String url;
}

File: src/main/kotlin/com/tencent/bk/devops/plugin/pojo/artifactory/PipelineBuildMaterial.kt
package com.tencent.bk.devops.plugin.pojo.artifactory

/**
 * 构建历史中的源材料
 */
data class PipelineBuildMaterial(
    val aliasName: String? = null,
    val url: String = "",
    val branchName: String? = null,
    val newCommitId: String? = null,
    val newCommitComment: String? = null,
    val commitTimes: Int? = null
)
    